scripts/build/Platform/Linux/build_config.json

```
@@ -387,12 +387,39 @@
       "CMAKE_TARGET": "install"
     }
   },
+  "install_20_release": {
+    "TAGS": [],
+    "PIPELINE_ENV": {
+      "NODE_LABEL": "ubuntu-20-packaging"
+    },
+    "COMMAND": "build_linux.sh",
+    "PARAMETERS": {
+      "CONFIGURATION": "release",
+      "OUTPUT_DIRECTORY": "build/linux_mono",
+      "CMAKE_OPTIONS": "-G 'Ninja Multi-Config' -DLY_MONOLITHIC_GAME=TRUE -DLY_PARALLEL_LINK_JOBS=4 -DLY_DISABLE_TEST_MODULES=TRUE -DLY_STRIP_DEBUG_SYMBOLS=TRUE",
+      "CMAKE_TARGET": "install"
+    }
+  },
+  "install_22_release": {
```

Can you add 'mono' to the pipeline names
```suggestion
  "install_20_mono_release": {
    "TAGS": [],
    "PIPELINE_ENV": {
      "NODE_LABEL": "ubuntu-20-packaging"
    },
    "COMMAND": "build_linux.sh",
    "PARAMETERS": {
      "CONFIGURATION": "release",
      "OUTPUT_DIRECTORY": "build/linux_mono",
      "CMAKE_OPTIONS": "-G 'Ninja Multi-Config' -DLY_MONOLITHIC_GAME=TRUE -DLY_PARALLEL_LINK_JOBS=4 -DLY_DISABLE_TEST_MODULES=TRUE -DLY_STRIP_DEBUG_SYMBOLS=TRUE",
      "CMAKE_TARGET": "install"
    }
  },
  "install_22_mono_release": {
```